cli/admin/README.md

```diff
@@ -97,27 +97,28 @@ All available commands:
 	cluster list-instances : Lists all nodes in the Pravega cluster (Controllers, Segment Stores).
 	config list : Lists all configuration set during this session.
 	config set <name=value list>: Sets one or more config values for use during this session.
+	container continuous-recover <number-of-runs> <seconds-between-runs>: Executes a local, non-invasive recovery for all SegmentContainers in the cluster during the specified duration.
 	container flush-to-storage <container-id> <segmentstore-endpoint>: Persist the given Segment Container into Storage.
 	container recover <container-id>: Executes a local, non-invasive recovery for a SegmentContainer.
-	container continuous-recover <container-id>: Executes a local, non-invasive recovery for all SegmentContainers in the cluster during the specified duration.
 	controller describe-readergroup <scope-name> <readergroup-id>: Get the details of a given ReaderGroup in a Scope.
 	controller describe-scope <scope-name>: Get the details of a given Scope.
 	controller describe-stream <scope-name> <stream-name>: Get the details of a given Stream.
 	controller list-readergroups <scope-name>: Lists all the existing ReaderGroups in a given Scope.
 	controller list-scopes : Lists all the existing scopes in the system.
 	controller list-streams <scope-name>: Lists all the existing Streams in a given Scope.
-	controller-metadata get <qualified-table-segment-name> <key> <segmentstore-endpoint>: Get the controller metadata entry for the given key in the table.
+	controller-metadata get <qualified-table-segment-name> <key> <segmentstore-endpoint> <json-file[OPTIONAL]>: Get the controller metadata entry for the given key in the table.
```

Comment:
Description could perhaps be :
`Get the value for the specified key from the specified ControllerMetadata Table.`

A common convention for specifying optional arguments to a command is to enclose them in square brackets (this is followed by almost all unix and windows shells). So your command documentation should look like:
```
controller-metadata get <qualified-table-segment-name> <key> <segmentstore-endpoint> [json-file]
```

Reply:
Changed it accordingly.